akka-remote/src/main/scala/akka/remote/artery/Codecs.scala

```diff
@@ -55,6 +56,12 @@ private[remote] class Encoder(
   val out: Outlet[EnvelopeBuffer] = Outlet("Artery.Encoder.out")
   val shape: FlowShape[OutboundEnvelope, EnvelopeBuffer] = FlowShape(in, out)
 
+  /**
+   * INTERNAL API
+   * Creates [[RemoteInstrument]] instances for use in this Encoder.
+   */
+  def createInstruments(system: ExtendedActorSystem): Vector[RemoteInstrument] = RemoteInstruments.create(system)
```

Comment:
Why is there a redirection here? Why not call `RemoteInstruments.create` directly in the code?